Update storage-use-azcopy-linux.md

richeney opened the pull request:
Updated dotnet to 2.0.0 as the latest version recommended on the https://www.microsoft.com/net/core#linuxubuntu page. Also added paragraph showing lsb_release -a command to help user determine which release of Ubuntu they are using so that they can navigate to the right section.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

richeney committed:
Update storage-use-azcopy-linux.md

Updated dotnet to 2.0.0 as the latest version recommended on the https://www.microsoft.com/net/core#linuxubuntu page. Also added paragraph showing lsb_release -a command to help user determine which release of Ubuntu they are using so that they can navigate to the right section.
